### PR TITLE
Aop

### PR DESCRIPTION
**Pull request by shuyangzhou**

### CODE REVIEW

**Comment by shuyangzhou** [comment]
ci:test

**Comment by shuyangzhou** [comment]
ci:test

**Comment by shuyangzhou** [comment]
ci:test

File: portal-impl/src/com/liferay/portal/spring/aop/ServiceBeanAutoProxyCreator.java [hunk 1]
```diff
@@ -62,8 +63,7 @@ protected Object wrapIfNecessary(
 
 		ProxyFactory proxyFactory = new ProxyFactory();
 
-		evaluateProxyInterfaces(beanClass, proxyFactory);
-
+		proxyFactory.setInterfaces(ReflectionUtil.getInterfaces(bean));
```
Comment (Preston-Crary) [inline portal-impl/src/com/liferay/portal/spring/aop/ServiceBeanAutoProxyCreator.java]: ReflectionUtil.getInterfaces() doesn't have the same result as spring's lookup.  Spring also includes extended interfaces.

**Comment by shuyangzhou** [comment]
ci:test

**Pull request closed without merging**